apps/fluent-tester/src/TestComponents/Common/TestProps.tsx

```diff
@@ -0,0 +1,23 @@
+import { Platform } from 'react-native';
+// Please read http://93days.me/testing-react-native-application/ to understand why we set accessibilityLabel here.
+
+/**
+ * The underlying Android test drivers use accessibilityLabel to find elements. However, all other platforms use testID.
+ * Because of this, we're adding the testProps function that takes in a unique identifier and returns the correct prop based on the platform
+ * (testID for Win32, Windows, macOS, iOS; accessibilityId for Android).
+ *
+ * If explict accessibilityLabel is being used for other platforms, apply testProps after it to override it for Android.
+ * @param id The string to be used as the test identifier
+ * @returns an object with the correct test identifier prop based on platform
+ */
+export function testProps(id) {
+  if (Platform.OS === 'android') {
+    return {
+      accessible: true,
+      accessibilityLabel: id,
+    };
+  }
+  return {
+    testID: id,
+  };
```

Comment:
Minor nit: this can be re-written to use the [`Platform.select()`](https://reactnative.dev/docs/platform#select) method:
```ts
return Platform.select({
  android: {
    accessible: true,
    accessibilityLabel: id,
  },
  default: {
    testID: id
  }
})
```
Feel free to disregard this.